ext/openssl/openssl.stub.php

```diff
@@ -129,6 +129,26 @@ function openssl_pkcs7_decrypt(string $infilename, string $outfilename, $recipce
 
 function openssl_pkcs7_read(string $infilename, &$certs): bool {}
 
+function openssl_cms_verify(string $filename, int $flags = 0, string $signerscerts = UNKNOWN, array $cainfo = UNKNOWN, string $extracerts = UNKNOWN, string $content = UNKNOWN, string $pk7 = UNKNOWN, string $sigfile = UNKNOWN, $encoding = OPENSSL_ENCODING_SMIME ): bool {}
```

Comment:
I've just seen in the RFC that some parameters have an `UNKNOWN` default value. Can you please define a regular default value for them? We are currently trying to get rid of `UNKNOWN` values from everywhere where it's possible.

Besides, if I saw correctly, `$signerscerts` is nullable (because of the `!` in ZPP), so this stub should also reflect that. That said, a `null` default value seems appropriate.

Let me know if you need help in this regard.

Reply:
I do not think this should be a problem.  I will let you know if I need help. Thank you for the offer.

Reply:
Sounds good! Thanks in advance :)

Reply:
Just to confirm, this looks like I should be changing args from something like...
`string $signercerts = UNKNOWN` 
to
`?string $signercerts = null`

Sound about right?

Reply:
Exactly! By default, we use `null` default values, but sometimes a non-nullable default can also be appropriate. So it's up to your consideration, (but generally, `null` is the right choice).

Reply:
ok.  I've done this, then.  They were all assumed to be nullable anyway.